src/AutoRest.CSharp/Common/Output/Builders/ClientBuilder.cs

```diff
@@ -85,6 +85,11 @@ public static IEnumerable<PagingMethod> BuildPagingMethods(OperationGroup operat
                     RestClientMethod method = restClient.GetOperationMethod(serviceRequest);
                     RestClientMethod? nextPageMethod = restClient.GetNextOperationMethod(serviceRequest);
 
+                    if (method == nextPageMethod)
```

Comment:
This check doesn't seem to be correct at first look. Why do we need to skip methods where get next is the same as get first?